src/twisted/conch/test/test_session.py

```diff
@@ -429,16 +446,17 @@ class SessionInterfaceTests(RegistryUsingMixin, TestCase):
     if not cryptography:
         skip = "cannot run without cryptography"
 
-    def setUp(self):
+    def setUp(self, register_adapters=True):
```

Comment:
I guess that this is ok... but maybe we can use this opportunity to remove the setup usage...so that later we don't have to revert it in test_setEnvNotProvidingISessionSetEnv.

My suggestion is to rename it to `getRegisteredSession()` or something like this and stop using self.session ... but there are many tests using self.session ... so maybe this can be done in a follow up cleanup...I can do it.